stable/rocketchat/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if include "host" . -}}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.image }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        env:
+        - name: MONGO_URL
+        {{ if .Values.mongodb.mongodbUsername }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "fullname" . }}
+              key: mongo-uri
+        {{ else }}
+          value: mongodb://{{ template "mongodb.fullname" . }}:27017/meteor
```

Comment:
To increase consistency with other Rocket.Chat documentation maybe use rocketchat as the db name here?